### DISCUSSION TITLE
What is the status of "Pass command line arguments for "Run Without Debugging"" ?

### DISCUSSION THREAD

**Discussion by Adnn** [microsoft/vscode-cmake-tools#3571] [General]
I am starting to use CMake tools, while developpins applications taking command line arguments.

I would like to run them without debugger (CMake Tools recognizes this is an important use case, as "Run Without Debugging" receives as much screen space as "Debug" in the different views), but I am not sure how to do it while providing command line arguments?

I found [this issue](https://github.com/microsoft/vscode-cmake-tools/issues/924), now closed, which concludes the discussion with a now-dead link.

What is the current status for this feature?

**Comment by gcampbell-msft** [comment]
One way to do this, and it may be the most supported and suggested way, is to create a launch.json. Then, you can select that in the debug pane dropdown, then in the top bar of VS Code, select "Run" -> "Run Without Debugging". 

![image](https://github.com/user-attachments/assets/e22b608f-728d-4558-a151-dbe85474a570)
